assets/js/blocks/price-filter/block.js

```diff
@@ -116,9 +132,16 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 					maxConstraint={ maxConstraint }
 					minPrice={ min }
 					maxPrice={ max }
-					step={ 10 }
-					currencySymbol={ CURRENCY.symbol }
-					priceFormat={ CURRENCY.priceFormat }
+					step={ 10 ** priceRangeData.currency_minor_unit }
```

Comment:
I think all currency and calcs should be handled by default in the PriceSlider if the props are not provided.   That will eliminate the need for any currency information inside this `PriceFilterBlock` component and values can remain stored internally as subunits.